fixes #24386 grpc service logging in console. (#24503)
dotnet/AspNetCore.Docs: aspnetcore/tutorials/grpc/grpc-start.md

      Request finished in 78.32260000000001ms 200 application/grpc
```

> if the gRPC service console do not have the logs then update the file appsettings.Development.json by adding the below two lines

Reviewer: Update the *appsettings.Development.json* file by adding the following lines:

> if the gRPC service console do not have the logs then update the file appsettings.Development.json by adding the below two lines

Reviewer: ```suggestion
Update the *appsettings.Development.json* file by adding the following lines:
```
